Sealer/sealer.cc

```diff
@@ -705,24 +710,29 @@ bool operator<(const FastaRecord& a, const FastaRecord& b)
 void findFlanks(FastaRecord &record,
 	int flanklength,
 	unsigned &gapnumber,
-	map<FastaRecord, map<FastaRecord, Gap> > &flanks)
+	map<FastaRecord, map<FastaRecord, Gap> > &flanks,
+	bool lower)
 {
 	const string& seq = record.seq;
+	std::string gap = "Nn";
```

Comment:
This can be a `const std::string`. You can initialize it with a ternary operator, like `const std::string gap(lower ? "..." : "...")`.